api/v1/Game.php

```diff
@@ -805,15 +803,11 @@ public function GetGameDetails(): array|PromiseInterface
             }
 
             $realtimePerEra = explode(",", $state["game_planning_era_realtime"]);
-            // todo: division by zero.
-            $currentEra = intval(floor($state["game_currentmonth"] / $state["game_planning_gametime"]));
-            $realtimePerEra[$currentEra] = $state["game_planning_realtime"];
-            $secondsPerMonthCurrentEra = round($state["game_planning_realtime"] / $state["game_eratime"]);
-            $monthsRemainingCurrentEra = $state["game_eratime"] - $state["game_planning_monthsdone"];
-            $totalRemainingTime = $monthsRemainingCurrentEra * $secondsPerMonthCurrentEra;
+            $totalRemainingTime = intval($state["game_planning_realtime"]);
+            $currentEra = intval($state["game_currentmonth"] / $state["game_eratime"]);
```

Comment:
Also the floor() function isn't necessary as intval() does all we need to get the right int

Reply:
alright